### ISSUE TITLE
[2.2-rc3] tsdb uid is broken, can't create new metrics

### ISSUE THREAD

**Issue by rluta** [OpenTSDB/opentsdb#629]
I'm running OpenTSDB 2.2 rc3 deb package on Ubuntu 14.04 configured with sharing.width=1 and random_uids.

I have inconsistent results from `tsdb uid`:

```
$ tsdb uid grep metrics test
metrics test.daily.bps: [-12, 48, 125]
metrics test.daily.bytes: [-30, -45, -2]
metrics test.daily.pkts: [16, -85, 108]
metrics test.daily.pps: [-77, -112, 96]

$ tsdb uid assign metrics test.new
Exception in thread "main" java.lang.IllegalArgumentException: No enum constant net.opentsdb.uid.UniqueId.UniqueIdType.metrics
	at java.lang.Enum.valueOf(Enum.java:236)
	at net.opentsdb.uid.UniqueId$UniqueIdType.valueOf(UniqueId.java:59)
	at net.opentsdb.tools.UidManager.assign(UidManager.java:364)
	at net.opentsdb.tools.UidManager.runCommand(UidManager.java:162)
	at net.opentsdb.tools.UidManager.main(UidManager.java:127)

$ tsdb uid assign METRIC test.new
METRIC test.new: [1, 4, 11]

$ tsdb uid grep METRIC test.new
METRIC test.new: [1, 4, 11]

$ tsdb uid grep metrics test
metrics test.daily.bps: [-12, 48, 125]
metrics test.daily.bytes: [-30, -45, -2]
metrics test.daily.pkts: [16, -85, 108]
metrics test.daily.pps: [-77, -112, 96]
```

In HBase, metric names created with METRIC actually appear in column 'name:METRIC', other metrics appear in column 'name:metrics'

**Comment by manolama** [comment]
Arg yeah, I goofed that up. Patch coming. Notes on your PR too. Thanks for finding and fixing!

**Comment by manolama** [comment]
Fixed in fb255884470597b3b4e8220b4250d75630099cdb
